content/guides/testing-strategies/working-with-graphql.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Working with GraphQL
+---
+
+<Alert type="info">
+
+## <Icon name="graduation-cap"></Icon> What you'll learn
+
```

Comment:
I would suggest calling out the required knowledge like `intercept`.

Reply:
@mccataldo I do not see `cy.intercept` as required knowledge, but something you will learn to use throughout this guide, applied to GraphQL.